daprdocs/content/en/developing-applications/building-blocks/service-invocation/service-invocation-overview.md

```diff
@@ -10,18 +10,20 @@ description: "Overview of the service invocation API building block"
 
 Using service invocation, your application can reliably and securely communicate with other applications using the standard [gRPC](https://grpc.io) or [HTTP](https://www.w3.org/Protocols/) protocols.
 
-In many environments with multiple services that need to communicate with each other, developers often ask themselves the following questions:
+In many microservice-based applications multiple services need the ability to communicate with one another. This inter-service communication requires that application developers handle inter-service problems like:
 
-- How do I discover and invoke methods on different services?
-- How do I call other services securely with encryption and apply access control on the methods?
-- How do I handle retries and transient errors?
-- How do I use tracing to see a call graph with metrics to diagnose issues in production?
+- Service discovery. How do I discover my different services?
```

Comment:
Trying to find a way to make this stand out more... maybe have the first sentence in the bullet point be bold and the question be italic? Or one or the other? What do you think?

Reply:
Agreed - good suggestion. I bolded the statement part and kept the questions themselves non-bolded. I think this helps make this part pop a bit more.